app/controllers/polls_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PollsController < ApplicationController
+  #user is at picture page, route meals#setup
+  def create
+    @poll = Poll.new(params.require(:poll).permit(:score, :meal, :cuisine))
+    @poll.user = current_user
+    raise
+    #current implementation assume whole poll is on one page
+    if @poll.save
+      redirect_to result_path
```

Comment:
Should not redirect to result page coz there are 10 polls to complete before going to result page